presto-product-tests/conf/presto/etc/jvm.config

```diff
@@ -13,7 +13,7 @@
 -XX:+CMSClassUnloadingEnabled
 -XX:+AggressiveOpts
 -XX:+HeapDumpOnOutOfMemoryError
--XX:OnOutOfMemoryError=kill -9 %p
+-XX:+ExitOnOutOfMemoryError
```

Comment:
This change means we require 8u92 for development, so we should also update the README and minimum version in the root POM.